bin/setup

```diff
@@ -24,7 +24,7 @@ Dir.chdir APP_ROOT do
   ]
 
   puts '== Setting up config overrides =='
-  File.write('config/application.yml', 'development:') unless File.exists?('config/application.yml')
+  File.write('config/application.yml', "development:\n  config_key:") unless File.exists?('config/application.yml')
```

Comment:
what about making it as an object and `.to_yaml`ing it instead of worrying about newlines etc?

```suggestion
  require 'yaml'
  default_application_yml = { development: { config_key: nil } }
  File.write('config/application.yml', default_application_yml.to_yaml) unless File.exists?('config/application.yml')
```

Reply:
much safer 😛